analyses/tmb-compare-tcga/TMB_d3b_code/README.md

```diff
@@ -0,0 +1,50 @@
+## TMB analysis
+
+###### Authors : Teja Koganti for D3B
+
+This analysis computes tumor mutation burden for different disease types.
+The first script takes single MAF file and filters variants  based filtering strategies from [Friends of Cancer research](https://jitc.bmj.com/content/8/1/e000147#DC1). TMB is computed  based on
+`(filtered variant counts* 1000000) / target BED length`
+
+### Calculate TMB
+  1. Reads the MAF file and filters based on variant_classification column
+
+  2. Reads in config file and defines metadata specific fields/columns and variants  list etc.
+
+  3. Reads in a target config file and determines `experimental strategy` and `cohort` from metadata file. MAF  variants  are filtered within this target file and the BED length is calculated
+
+  4. Calculates TMB
+      - Calculates TMB based on `((# of variants)*1000000) / size of BED)`
```

Comment:
```suggestion
      - Calculates TMB based on `((# of nonsynonymous variants based on FoCR criterion within the BED ranges)*1000000) / size of BED)`
```
This may not be the best way to make this specification, but I want to make sure my understanding is correct.